app/controllers/appointment_summaries_controller.rb

```diff
@@ -31,13 +33,11 @@ def preview
   def appointment_summary_params
     params
       .require(:appointment_summary)
-      .permit(:title, :first_name, :last_name,
-              :date_of_appointment,
-              :guider_name, :guider_organisation,
-              :address_line_1, :address_line_2, :address_line_3, :county, :town, :postcode, :country,
-              :has_defined_contribution_pension,
-              :supplementary_benefits, :supplementary_debt,
-              :supplementary_ill_health, :supplementary_defined_benefit_pensions, :format_preference,
-              :appointment_type)
+      .permit(*appointment_summary_editable_column_names)
+      .merge(user: current_user)
+  end
+
+  def appointment_summary_editable_column_names
```

Comment:
I thought about that, but this feels more like a view/controller concern than a model concern.

Should the model really need to know the list of fields that are editable in the view?